module/Database/view/database/member/subscribe.phtml

```diff
@@ -62,33 +35,13 @@ $form->setAttribute('id', 'form-subscribe');
         <div class="col-md-9">
             <h1>
                 <span class="gewis-red">GEWIS</span> <?= $this->translate('Inschrijving') ?>
-                <?php if (DATABASE_REQUIRE_IBAN): ?>
-                    <?= $this->translate('SEPA-machtiging') ?>
-                <?php endif; ?>
             </h1>
             <p>
-                <?= sprintf(
-                    '%s %s',
-                    $this->translate('Welkom! Met dit formulier kun je je inschrijven voor een
+                <?= $this->translate('Welkom! Met dit formulier kun je je inschrijven voor een
 lidmaatschap van GEWIS. Het lidmaatschap is geldig zo lang je studeert en kost
-eenmalig 15 euro.'),
-                    DATABASE_REQUIRE_IBAN ? $this->translate('Deze betaling zal middels een eenmalige SEPA-machtiging
-verlopen aan GEWIS. De adresgegevens van GEWIS zijn als volgt:') : ''
-                )?>
+eenmalig 15 euro.') ?>
```

Comment:
Add more information on that submitting the form means that you are brought directly to the checkout page.

Reply:
Changed.